### PR TITLE
[AVS] Add feature toggle, auth guard and mocks

### PR DESCRIPTION
**Pull request by acrollet**
## Summary

Adds a global feature toggle, requires login and adds a basic container (+ unit tests) for AVS. Components for the contents will come in a follow-on.

### CODE REVIEW

File: src/applications/avs/api/mocks/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (acrollet) [reply src/applications/avs/api/mocks/index.js:1]: feature toggle

File: src/applications/avs/api/mocks/avs/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const data = id => {
+  let avs = {};
+  const filename = `../../../tests/fixtures/${id}.json`;
+
+  try {
+    // eslint-disable-next-line import/no-dynamic-require
```
[the remark this answers is not in the document]
Comment (acrollet) [reply src/applications/avs/api/mocks/avs/index.js:6]: allows dynamically loading different mocks

File: src/applications/avs/api/mocks/avs/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    avs = require(filename);
+  } catch (e) {
+    // eslint-disable-next-line no-console
```
[the remark this answers is not in the document]
Comment (acrollet) [reply src/applications/avs/api/mocks/avs/index.js:9]: in mock only

**Review by powellkerry — APPROVED** [review]

**Review by mdewey — APPROVED** [review]
LGTM! I see a bunch of patterns from the check in app here. We should probably look to socialize these into more apps

**Pull request merged**